controllers/error-handlers.controller.js

```diff
@@ -0,0 +1,13 @@
+exports.psqlErrorHandler = (err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "invalid parameter" });
+  } else {
+    next(err);
+  }
+};
+
+exports.customErrorHandler = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  }
```

Comment:
Will need   } else {
    next(err)
  } 
so that any errors not matching any conditions will be passed to the next error handler.